README.md

```diff
@@ -66,6 +66,7 @@ Give a ⭐️ if you like this project!
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.